.github/workflows/conformance_with_build.yml

```diff
@@ -35,22 +35,22 @@ jobs:
             api-gateway-image: "consul-api-gateway:local-build"
             consul-image: "hashicorp/consul:1.11"
             envoy-image: "envoyproxy/envoy:v1.20-latest"
-            consul-k8s-version: "v0.49.0"
+            consul-k8s-version: "Consul-api-gateway-create-namespaces"
```

Comment:
This can be removed I imagine since we wound up not needing the RBAC changes we were initially discussing